example/FILTER.xprof/Makefile

```diff
@@ -24,4 +24,4 @@ params.o: params.c
 	$(CC) params.c -I $(METABABEL_ROOT)/include $(CFLAGS) $(CFFLAGS_BT2) -c
 
 clean:
-	rm -f *.o *.so
+	rm -f *.o *.so component.h create.c create.h dispatch.c dispatch.h params.c xprof.c
```

Comment:
Not sure I Like that; clean should not delete generated code.
Maybe add a new `super_clean` rules?